Revit_Engine/Convert/Aliases.cs

```diff
@@ -36,8 +36,9 @@ public partial class Convert
         public const string CategoryName = "Revit_categoryName";
         public const string ViewName = "Revit_viewName";
         public const string ViewTemplate = "View Template";
-        public const string Edges = "Revit_edges";        
+        public const string Edges = "Revit_edges";
         public const string Surfaces = "Revit_surfaces";
+        public const string Representation = "MeshRepresentation";
```

Comment:
Assuming this is returning a mesh, wouldn't this be `Mesh = Revit_mesh` following the convention of the other names?

Reply:
This change came from @alelom who wants representation be software agnostic (as it does not necessarily need to come from Revit).

However, this brings another issue being Revit and Representation fragments - I am happy to implement the former, @alelom how do you feel about taking care of the latter? Looks like you are best positioned to that.